src/VisualStudio/Core/Def/Implementation/LanguageClient/AbstractLanguageServerClient.cs

```diff
@@ -68,8 +68,6 @@ internal abstract class AbstractLanguageServerClient : ILanguageClient
 
         public Task<Connection> ActivateAsync(CancellationToken token)
         {
-            Contract.ThrowIfFalse(_languageServer == null, "This language server has already been initialized");
```

Comment:
Discussed offline - this contract has been useful to catch weird activation bugs.  Current idea is to have a flag on the language server indicating if it has been shutdown and check that instead.  We should throw if shutdown/exit has not been called on the language server.

Once _languageServer is re-assigned the previous language server instance should be gc'd